src/webview/tests.py

```diff
@@ -36,6 +36,267 @@
+        self.user = User.objects.create_user(username="admin", password="pw")
+        self.user.is_staff = True
+        self.user.save()
+
+        # Create a GitHub social account for the user
+        SocialAccount.objects.get_or_create(
+            user=self.user,
+            provider="github",
+            uid="123456",
+            extra_data={"login": "admin"},
+        )
+
+        self.client = Client()
+        self.client.login(username="admin", password="pw")
+
+        # Create CVE and related objects
+        self.assigner = Organization.objects.create(uuid=1, short_name="foo")
+        self.cve_record = CveRecord.objects.create(
+            cve_id="CVE-2025-0001",
+            assigner=self.assigner,
+        )
+        self.description = Description.objects.create(value="Test description")
+        self.metric = Metric.objects.create(format="cvssV3_1", raw_cvss_json={})
+        self.affected_product = AffectedProduct.objects.create(
+            package_name="dummy-package"
+        )
+        self.affected_product.versions.add(
+            Version.objects.create(status=Version.Status.AFFECTED, version="1.0")
+        )
+        self.cve_container = self.cve_record.container.create(
+            provider=self.assigner,
+            title="Dummy Title",
+        )
+        self.cve_container.affected.add(self.affected_product)
+        self.cve_container.descriptions.add(self.description)
+        self.cve_container.metrics.add(self.metric)
+
+        # Create maintainer and metadata
+        self.maintainer = NixMaintainer.objects.create(
+            github_id=123,
+            github="testuser",
+            name="Test User",
+            email="test@example.com",
+        )
+        self.meta1 = NixDerivationMeta.objects.create(
+            description="First dummy derivation",
+            insecure=False,
+            available=True,
+            broken=False,
+            unfree=False,
+            unsupported=False,
+        )
+        self.meta1.maintainers.add(self.maintainer)
+
+        self.meta2 = NixDerivationMeta.objects.create(
+            description="Second dummy derivation",
+            insecure=False,
+            available=True,
+            broken=False,
+            unfree=False,
+            unsupported=False,
+        )
+        self.meta2.maintainers.add(self.maintainer)
+
+        # Create evaluation and derivations
+        self.evaluation = NixEvaluation.objects.create(
+            channel=NixChannel.objects.create(
+                staging_branch="release-24.05",
+                channel_branch="nixos-24.05",
+                head_sha1_commit="deadbeef",
+                state=NixChannel.ChannelState.STABLE,
+                release_version="24.05",
+                repository="https://github.com/NixOS/nixpkgs",
+            ),
+            commit_sha1="deadbeef",
+            state=NixEvaluation.EvaluationState.COMPLETED,
+        )
+
+        # Create two derivations for the same package
```

Comment:
```suggestion
```

artefact or typo? the two derivations are simply being linked to the same suggestion